Automation/StorageAccount/create_Azure_StorageAccount.ps1

```diff
@@ -0,0 +1,66 @@
+param(
+    [string]
+    [Parameter(Mandatory=$true)]
+    $resourceGroup, # Name of resource group to put storage account in
+
+    [string]
+    [Parameter(Mandatory=$true)]
+    $location, # Regional location EG "westus2" or "japaneast"
+
+    [string]
+    [Parameter(Mandatory=$true)]
+    $storageAccountName, # Name to give storage account
+
+    # Below required for Login module
+    [string] [Parameter(Mandatory=$true)] $tenantId,
+    [string] [Parameter(Mandatory=$true)] $applicationId,
+    [string] [Parameter(Mandatory=$true)] $secret,
+    [string] [Parameter(Mandatory=$true)] $subscriptionId
+)
+
+Start-Transcript -Path "$PSScriptRoot\create_Azure_StorageAccountlog.log"
+Write-Host "Logging to $PSScriptRoot\create_Azure_StorageAccountlog.log"
+Write-Host "Create Azure Storage Account"
+
+# Custom module Login
+Import-Module ..\Login
+# Provides function Login (tenantID, applicationID, secret, subscriptionID)
+Login $tenantId $applicationId $secret $subscriptionId
```

Comment:
I used the Connect-AzAccount command, didnt know there was this Login module too. Works great.

Reply:
Yeah, this is brand new! It should be a very quick fix to add it to all the scripts. Especially since it is just some logic that takes those four keys and runs the `Connect-AzAccount` cmdlet using service principals

Reply:
good to know Logi-in module